core/src/components/range/range.tsx

```diff
@@ -629,7 +636,7 @@ Developers can dismiss this warning by removing their usage of the "legacy" prop
               'label-text-wrapper-hidden': !this.hasLabel,
             }}
           >
-            <slot name="label"></slot>
+            {label !== undefined ? <div class="label-text">{label}</div> : <slot name="label"></slot>}
```

Comment:
Nit: Why do we not render `.label-text` when using the slot?

In other words, why not do `<div class="label-text">{ label === undefined ? <slot name="label"></slot> : label }</div>`

Reply:
For consistency with other components. Components that use a prop wrap it in `.label-text` (e.g. [input](https://github.com/ionic-team/ionic-framework/blob/FW-3669/core/src/components/input/input.tsx#L587)) while those that use a slot don't (e.g. [checkbox](https://github.com/ionic-team/ionic-framework/blob/FW-3669/core/src/components/checkbox/checkbox.tsx#L268)). I can add the wrapper for both cases here if desired, though.

Reply:
You can leave it as is if you'd like. Main reason I ask is I'm going to need to do something similar with my `ion-select` label slot work, so would be good to be consistent. I'll make sure my PR follows what you're doing here.